src/env.ts

```diff
@@ -216,7 +216,7 @@ const envDefinitions = {
     desc: "enable experimental support of Bluetooth",
   },
   EXPERIMENTAL_CURRENCIES: {
-    def: "",
+    def: "cardano",
```

Comment:
Please don't add the coin in the default value, it is meant to stay empty and only be set at runtime.